netlify.toml

```diff
@@ -7,7 +7,7 @@ functions = "functions"
 command = "make non-production-build"
 
 [build.environment]
-HUGO_VERSION = "0.52"
+HUGO_VERSION = "0.49"
 
 [context.production.environment]
```

Comment:
@lucperkins, can you further explain why the HUGO_VERSION in netlify.toml is remaining at 0.49? Who/what calls the non-production-build target (why should version 0.49 vs 0.52 matter)?

Reply:
Good catch. This should indeed also be 0.52. Fixed.